packages/migration/src/migrations/events/1746712582483_create_events_and_actions_tables.sql

```diff
@@ -158,4 +158,4 @@ DROP TYPE IF EXISTS action_status;
 
 DROP TABLE IF EXISTS events;
 
-DROP TABLE IF EXISTS locations;
+DROP TABLE IF EXISTS locations;
```

Comment:
Accidental edit?

Reply:
Yes, from merges, and then prettier formatted the file on save